.github/workflows/version-or-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Version or Publish
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  changesets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+
+      - name: Setup Node.js 12.x
```

Comment:
Shouldn't we use at least node 14?

Reply:
I'm not sure if that has changed since then but apparently noode 14 couldn't be used a few months back:
https://github.com/actions/github-script/pull/182#issuecomment-903966153

Reply:
Where is actions/github-script being used?

Reply:
ah, i might have confused both things in a hurry - it might only not be possible to use node 14 for action itself but the setup-node action can actually setup node 14